mmaction/models/detectors/fast_rcnn.py

```diff
@@ -0,0 +1,249 @@
+import numpy as np
+import torch
+import torch.nn as nn
+from mmdet.core import bbox2roi
+
+from mmaction.core.bbox import bbox2result, build_assigner, build_sampler
+from .. import builder
+from ..registry import DETECTORS
+from .base import BaseDetector
+
+
+@DETECTORS.register_module
+class FastRCNN(BaseDetector):
```

Comment:
Is it possible to use mmdet directly?

Reply:
A lot of refactoring work will be needed, maybe in the next version. (Currently, the FastRCNN is implemented with minimal features).

Reply:
Could you elaborate on the "a lot" part? Let's see if it is worthwhile